compiler/rustc_hir_analysis/src/check/compare_impl_item.rs

```diff
@@ -1215,7 +1215,7 @@ fn check_region_late_boundedness<'tcx>(
         return None;
     };
 
-    let errors = ocx.select_where_possible();
+    let errors = ocx.try_evaluate_obligations();
     if !errors.is_empty() {
         return None;
```

Comment:
this is the only `try_evaluate_obligations` in this function and it feels odd :> ah well